src/config/default.yml

```diff
@@ -111,124 +112,12 @@ try:
   #  - scopes: scopes for the task graph
   #  - level: SCM level (defaults to 1) passed to the decision task
   projects:
-    # Try is unique in that it parses the commit message
-    try: # Note the keys match the "alias" which treeherder defines.
-      url: "{{{host}}}{{{path}}}/raw-file/{{revision}}/testing/taskcluster/tasks/decision/try.yml"
-      level: 1
-      scopes:
-        - "assume:repo:hg.mozilla.org/try:*"
-
-    # we don't create task graphs for gaia (gaia-taskcluster does that), but we do handle
-    # retriggers for gaia.  For that, we use the gaia role (which is not a Mozilla tree)
-    gaia:
-      scopes:
-        - "assume:repo:github.com/mozilla-b2g/gaia:*"
-
-    gaia-master:
-      scopes:
-        - "assume:repo:github.com/mozilla-b2g/gaia:*"
-
-    # same story as gaia
+    # Note that most of this data is loaded from the productionBranchesUrl.
+    # These are non-gecko, legacy projects that are not included there.
     bmo-master:
```

Comment:
I think it is/was retriggering?  Maybe treeherder?  Anyway, when we are ready to kill m-t we can tackle that.